src/Components/ArtworkFilter/ArtworkFilters/ArtistsFilter.tsx

```diff
@@ -109,14 +109,13 @@ export const ArtistsFilter: FC<ArtistsFilterProps> = ({ expanded, fairID }) => {
 
   const isFollowedArtistCheckboxSelected =
     !!user && includeArtworksByFollowedArtists
-  const followedArtistArtworkCount = filterContext?.counts?.followedArtists ?? 0
   const hasSelection = artistIDs.length > 0 || isFollowedArtistCheckboxSelected
 
   return (
     <FilterExpandable label={label} expanded={hasSelection || expanded}>
       <Flex flexDirection="column" gap={2} mt={1}>
         <Checkbox
-          disabled={!followedArtistArtworkCount}
+          disabled={!user}
```

Comment:
🎉

Reply:
You could even do something like:

```tsx
{user && (
  <Checkbox ... 
)
```

Might be better to hide it altogether? But either way is 👍

Reply:
Eh, I'll keep it as-is. Very unobtrusive, and the disabled state is nice.